tests/app/data/test_schema_validation.py

```diff
@@ -135,7 +135,7 @@ def test_child_answers_define_parent(self):
     @staticmethod
     def all_schema_files():
         schema_files = []
-        for folder, _, files in os.walk(settings.EQ_SCHEMA_DIRECTORY):
+        for folder, _, files in os.walk(_get_schema_path()):
             for filename in files:
                 if filename.endswith(".json") and filename != 'schema_v1.json':
```

Comment:
If we move `schema_v1.json` to the `data` directory, then we shouldn't need this logic excluding it.
Also, if we do that, can this method be replaced with a call to `available_json_schemas`?